src/DevYeah.LMS/DevYeah.LMS.Data/paginate/IQueryableExtention.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Linq;
+
+namespace DevYeah.LMS.Data.paginate
```

Comment:
"paginate” -> "Pagination": Pascal casing, and should be a noun

Please align all the files for the same namespace